#  Protein Hydration
Water is a major contributor to a protein's 3-D structure and the protein controls the structuring of its surrounding water.

 

 

- The contribution of water to protein structure
- Water in protein recognition and binding 
- Water in protein and enzyme function 
- Protein folding and denaturation

 

'Interactions with water govern the folding, structure, stability and activity of proteins'

The contribution of water to protein structure
Protein hydration is very important for their three-dimensional structure, the dynamic ensemble of conformations [2249, 2921].and activity [472, 1093, 1345, 2005]. Structural determination with a resolutiion of about 1.5–1.6 Å is necessary to observe the continuous hydration layer at the protein surface [3049]. Fluctuations of the protein surface groups drive and are driven and controlled by the surrounding network of water molecules [2648, 2917]. Indeed, proteins lack biological activity in the absence of sufficient hydrating water (usually at least a monolayer covering; > 1.5 mols H2O mol-1 amino acid residue). High-resolution X-ray diffraction detects about one H2O mol-1 amino acid that are relatively static [2677]. The aqueous structuring around proteins is affected out to at least 1 - 1.5 nm from its surface or 2 - 3 nm between neighboring proteins, as shown by terahertz spectroscopy [1368, 2102] and molecular dynamics [2886], e with even small proteins (e.g. bovine serum albumin, 66,463 Da) affecting the whole of its unstirred (Nernst) layer of about 20,000 neighboring water molecules [2102]. f Certainly, the range of these interactions is governed by directed hydrogen bonding interactions that extend further than the electrostatics dependent Debye screening length (< 1 nm). Additionally, the presence of glycans attached to (glyco)proteins imposes a long-range order on the water structure out to several nanometers, dependent on the orientation of the glycan [2104]. Some water molecules interact with the surface, reorienting both themselves and the surface groups whereas other water molecules link these to the bulk in an ordered manner whilst remaining in dynamically active [1695]. In solution, proteins possess a conformational flexibility, which encompasses a wide range of hydration states, not seen in the crystal a or in non-aqueous environments. The equilibrium between these states will depend on the activity of the water within its microenvironment; that is, the freedom that the water has to hydrate the protein [434]. 

 

Protein conformations demanding greater hydration are favored by more (re-)active water (for example, high-density water containing many weak bent and/or broken hydrogen bonds) and 'drier' conformations are relatively favored by lower activity water (for example, low-density water containing many strong intra-molecular aqueous hydrogen bonds). Surface water molecules are held to each other most strongly by the positively-charged basic amino acids. The exchange of surface water (and hence the perseverance of the local clustering and the overall system flexibility) is controlled by the exposure of the groups to the bulk solvent (that is, greater exposure correlates to greater flexibility and freer protein chain movement) [975]. The structuring and dynamics of the inner hydration layers are affected by the protein surface chemical composition (hydrophobic, hydrogen bonding, polar, acidic and basic groups), the complex topography (clefts, grooves, pockets and protrusions), the presence of water 'wires' and the requirements of the secondary hydration layers' hydrogen bonding [2781]. Counter-intuitively, hydrating water is slower near hydrophobic patches than near hydrophilic residues. This is due to stronger confinement often found in these areas. Protein conformational fluctuations affect the hydration shell dynamics in a heterogeneous manner across the protein surface. They particularly modify and reduce the aqueous dynamics within concave areas of partial confinement [2725].

 

Hydration also affects the reactions and interactions of coenzymes and cofactors; thus, the various redox potentials (and hence whether they oxidize or reduce) of some iron-sulfur proteins are accounted for by differential hydration rather than direct protein binding effects [1344].Hydration of ionic solutes shields them from the protein surface, such that ionic strength changes have little effect on protein charge distribution [2278]. 

 

The folding of proteins depends on the same factors as control the junction zone formation in some polysaccharides; that is, the incompatibility between the low-density water (LDW) and the hydrophobic surface that drives such groups to form the hydrophobic core. b This drive for hydrophobic groups to mostly cluster away from the protein surface (in water soluble proteins) is controlled by the charged and polar group interactions with each other and their cooperative hydration [2716]. It is very difficult to quantitatively model this driving force [2491, 2492]. Dehydration of Concanavalin A induces the decrease of its β-sheet secondary structure due to breakage of their β-sheet-linking hydrogen -bonds [2889]. 

 

Interestingly, in pure water and in the absence of screening dissolved ions, some buffer-insoluble proteins are quite soluble due to the weak (here unshielded) interactions between the protein's intrinsic charges [1375]. Ions have effects on both protein solubility and protein stability (see the Hofmeister Series). Non-ionic kosmotropes, which stabilize low-density water, consequentially stabilize the structure of proteins. In addition, water acts as a lubricant [822], so easing the necessary peptide amide-carbonyl hydrogen bonding changes. The biological activity of proteins appears to depend on the formation of a 2-D hydrogen-bonded network spanning most of the protein surface and connecting all the surface hydrogen-bonded water clusters [978]. Such a water network is able to transmit information around the protein and control the protein's dynamics, such as its domain motions [977]. In particular, the translational diffusion of the interfacial water molecules stimulates the large-amplitude protein motions required for biological activity [2435]. Hydrating water molecules can control the functional motions of multi domain proteins. For example, each subunit in hexameric glutamate dehydrogenase (GDH) comprises a nucleotide-binding domain (N-domain) and a core domain (C-domain) for hexamer formation [2583]. Cooperative changes in the hydration structure within the cleft between the domains switches domain-closure on or off, with the hydrophobic closure necessary for substrate binding. Much life functions optimally at about 37 °C where a spanning hydration network is about to break down on heating, perhaps to help compensate for the biosystem's entropic changes [1541].

 

Intramolecular peptide (amide) hydrogen bonding makes a major contribution to protein structure and stability. Equilibrium H/D fractionation may be used to quantitatively assess hydrogen bond strengths and has shown them to be marginally stronger on average in α-helices rather than β-sheets by ~0.8 kJ ˣ mol-1 [2115]. However such hydrogen bonds are only effective in the absence of accessible competing water. Even just the presence of close-by water molecules causes peptide hydrogen bonds to lengthen [524], so loosening the structure. Water mediated hydrogen bonding between peptide links has been found to be particularly important in the structure of collagen where only a third of the peptide links directly hydrogen bond to other peptide links [1314] and the stability of the triple helix is sequence dependent [1438]. The single-water bridges found within the individual triple-helix collagen molecules have been found to be dynamically linked with freely exchangeable hydrogen atoms, rather than ice like [2840]. Water molecules can bridge between the carbonyl oxygen atoms and amide protons of different peptide links to catalyze the formation, and its reversal, of peptide hydrogen bonding as well as forming long-lived linkages stabilizing protein-ligand and protein-protein interfaces (for example, [688]). The internal molecular motions in proteins, necessary for biological activity, are very dependent on the degree of plasticizing, which is determined by the level of hydration [810 (greater water gives lower glass transition temperature and greater plasticity [2245]). Thus internal water enables the folding of proteins and is only expelled from the hydrophobic central core when finally squeezed out by cooperative protein chain interactions [352]. Many water molecules (similar in amount to individual amino acids) remain behind buried in the core of the proteins, so forming structurally-important hydrogen-bonded linkages. The position of the ESreversible arrowCS equilibrium around enzymes has been shown to be important for their activity with the enzyme balanced between flexibility (CS environment) and rigidity (ES environment). The addition of non-ionic chaotropes only, or kosmotropes only, may inhibit the activity of enzymes (by shifting the equilibrium to the right or left respectively) whereas an intermediate mixture of kosmotropes and chaotropes restores optimum activity [276]. Also, the ease of enzyme-substrate contact may be controlled by the level of water structuring in the protein environs [882]. Water in the crystal structure can be positioned by diffraction at high resolution (0.15 - 0.16 nm), with less being seen at lower resolutions [2554]. The crystal structures of proteins suggest that the commonest water polygons surrounding proteins are (H2O)3 trimers (typical of CS environments) and (H2O)5 pentamers (typical of ES environments) with lower amounts of tetramers, hexamers, and heptamers [1849].

 

The first hydration shell around proteins (~0.3 g/g) is ordered; with high proton transfer rates and well resolved time-averaged hydration sites; surface water showing coherent hydrogen-bond patterns with large net dipole fields [702]. The hydrogen bonds holding these water molecules to the protein are stronger with longer lifetimes than bulk water [1355] and this water is unavailable to colligative effects. This hydration shell, by retaining some liquidity, protects the protein against irreversible denaturing effects, such as very low temperatures [2338]. As hydration sites may be positioned close together and therefore mutually exclusive, it has been argued that the solvation is better described as a water distribution density function rather than by specific water occupied sites. For example, no more than 164 of the 294 high-density hydration sites around myoglobin are occupied at any moment and there is no correlation between the maximum site density, occupancy and residence time [542]. The first hydration shell is also 10-20% denser than the bulk water and probably responsible for keeping the molecules sufficiently separated so that they remain in solution [315] (that is, solutions are kinetically stable but often thermodynamically unstable). c Although a significant amount of this density increase has been shown to be due to simple statistical factors dependent upon the way that the surface is defined in depressions [401], much is due to a protein's structure with the excess of polar hydration sites (tending to increase surface density) d easily counteracting the remaining non-polar surface groups (tending to produce low-density surface water). This water is required for the protein to show its biological function as, without it, the necessary fast conformational fluctuations cannot occur. The interfacial water network shows significant cooperativity with the protein's global hydration shell that behaves collectively [2908]. Thus proteins have no activity (and enter a glassy state, at about 220 K) when the surrounding water becomes predominantly low-density [1197]. Also, dry proteins remain rigid (glassy) until sufficient water is added to hydrate the protein's charged groups, which occurs at a water weight fraction of 0.05 for lysozyme at 25 °C [2240].

 

Using X-ray analysis, the hydration shell shows a wide range of non-random hydrogen-bonding environments and energies. Proteins are formed from a mixture of polar and non-polar groups. Water is most well-ordered round the polar groups where residence times are longer, but where they will interfere with water's natural hydrogen bonding, than around non-polar groups where aqueous clathrate structuring may form [2578]. 

Tenebrio molitor antifreeze protein, from PDB  1EZG, binding to ice
 

Antifreeze proteins possess ice-binding sites that are typically flat, extensive and relatively hydrophobic [2819]. They introduces greater order (lower entropy) in their hydration shell for their enthalpic binding of the surface water molecules [3048]. This is most energetically favorable where there is no pre-existing order in the water that requires destruction and the surface water has dangling O-H groups available for the crystalline hydrogen bonding. Surfaces that raise the water activity above that of the bulk water are ideal for this purpose. Strongly hydrogen bonding surfaces (with low water activity) force surface water molecules into particular structures that have difficulty in fitting to the ice crystals.

 

Interestingly, the clathrate pentagonal structuring has been found to extensively surround the helices in the four-helix bundle winter flounder antifreeze protein, so retarding the formation of harmful ice crystals in fish living in near-freezing water [2084]. Rather than binding directly with ice, the antifreeze protein type three (AFP-III), from the Antarctic eelpout, arranges some of its surface water molecules into an ice-like array well above 0 °C but retained below 0 °C, which it then uses to bind to ice crystals [2801].

  

In the Tenebrio antifreeze protein there are tandem 12-residue repeats (TCTxSxxCxxAx) that form a β-helix with regularly spaced threonine residues (0.744 nm and 0.464 nm), each turn of the helix, that make a match to water molecules in the ice prism plane (0.738 nm and 0.452 nm (see right [2802]). A similar binding site has been found in the ice-binding protein of the Antarctic bacterium Marinomonas primoryensis that is used to anchor the organism to Antarctic ice flows where oxygen and nutrients are more available. The protein binds to sea ice through a flat, repetitive two-dimensional array of Thr and Asn residues.Both polar and non-polar groups create order in the water molecules surrounding them but their ability to do this and the types of ordering produced are very different. Polar groups are most capable of creating ordered hydration through hydrogen bonding and ionic interactions (an excellent guide to amino acid hydrogen bonding is given elsewhere). The ordering created in the water surrounding proteins extends the proteins' electrostatic surfaces well away from their physical (that is, amino acid) surfaces giving them far greater electrostatic visibility to visiting ligands [1156]. This non-specific electrostatic effect of the water effect is additional to any specific directed hydrogen bonding that may extend away into the bulk from the surface

 

Molar volumes of the amino acids 
 

The line is the best straight line from the amino acid data but is expected to lie on the low-density side of aqueous structuring.
Hydrogen bonding to the amino acid side-chains helps to determine the three-dimensional folding of the proteins. This particularly involves the acidic (aspartic acid, D; glutamic acid, E), basic (lysine, K; arginine, R) and hydrophilic amino acids (asparagine, N; glutamine, Q; serine, S; threonine T) but also involves the aromatic amino acids that can form π···H-O hydrogen bonds (histidine, H; phenylalanine, F; tyrosine, Y; tryptophan, W) [3069]. The graph opposite shows the molar volumes of the amino acids [1063] (using the one-letter code). Leucine (L) has the largest molar volume relative to that expected from its relative molecular mass (molecular weight) and forms low-density clathrate water structures if exposed to the solution. It is followed by isoleucine (I), valine (V), phenylalanine (F), lysine (K) and arginine (R). Aspartic acid (D) has the smallest molar volume relative to that expected from its molecular mass and forms higher density water structuring. It is followed by asparagine (N), glutamic acid (E), serine (S), cysteine (C), glutamine (Q), histidine (H), threonine (T) and glycine (G). The hydration properties of alanine (A), proline (P), methionine (M), tyrosine (Y) and tryptophan (W) are slightly kosmotropic. The relative hydrophobicities have been assessed as D<E<K<R<Q

<H<N<S<Y<W<T<M<F<C<G<A<I<V<L<P [2820].

 

The water is slow to exchange, showing the dynamic behavior of bulk water 25 °C colder [147]. Low-density water (such as ES) is promoted [148, 276] surrounding this dense hydration and polyelectrolyte double layer (as described in the 'Polysaccharide hydration' section). Non-polar groups promote clathrate structures [153] (such as ES) surrounded by denser water. It is no surprise, therefore that the degree of hydrophobic hydration is correlated with the hydration of the polar groups further away. As clathrate-type structures break down at higher temperatures, hydrophobic hydration shows greater temperature dependence than hydrophilic hydration [2761]. Clathrate shells contain loosely held water with greater rotational freedom than in the bulk [139]. However under favorable conditions, clathrate hydrophobic hydration may exert pressure on non-polar C-H bonds pushing them in, so contracting their bond length and increasing their vibrational frequency. This blue-shifting (that is, the vibration frequency increases and intensity reduces) 'push-ball' hydration [149] should not necessarily be thought of as 'typical' hydrogen bonding even if the CH···OH2 distances are suitably close (see [1293]). They can be considered as part of a continuum of hydrogen bonding behavior, however, where sometimes the OH2 behaves as a much more weakly interacting base than usual and the C-H behaves with reversed dipolar behavior compared with the more usual O-H hydrogen-bonding partners [625].

 

There are significant differences in the directional rates of water diffusion perpendicular and parallel to the protein surface that are maximal at about 6 Å but still determinable at 15 Å from the surface [542]. It is clear that evolutionary processes have made use of the organization in this water surrounding proteins to create preferred diffusive routes and orientation for metabolites and favored conformational changes and interactions. Such diffusive paths lead to binding sites with their own helpful hydration. It has been suggested that pressure waves formed from flickering water clusters (for example, as described elsewhere) may link protein molecular vibrations, so carrying information through the intracellular milieu [549], 2678] and powering product movement between enzymes in biochemical pathways [665]. 

Water in protein recognition and binding
barnase-barnstar protein, from PDB 1BRSWater molecules form an integral part of most protein-protein [1339], protein-DNA [1340] and protein-ligand [1341] interactions, aiding the mutual recognition and both the binding thermodynamics and binding kinetics [1338]. Molecular assembly is driven and steered by a maximizing of fulfilled hydrogen bonding and reduction in the number of unmet hydrogen- bonding opportunities (so-called 'frustration') at the protein-water interface [2843].

 

Generally, hydrophilic residues make up the protein−protein and protein−DNA binding surfaces. These interact via water molecules. The relative diameters of the binding cavity and the binding ligand determine the kinetics and thermodynamics of (hydrophobic) binding cavity-ligand systems. This binding involves a sharp (cooperative) dewetting transition on their formation and a continuous dewetting transition on the loss of the ligand [2575]. Removal of water from the binding site (before ligand binding) may be assisted by non-polar ligand groups due to a reduction in the interference of the previously-bound water with the protein's internal hydrogen bonding and increased bulk hydrogen bonding [2922].

 

The water at the interface of the tightly bound barnase-barnstar enzyme- inhibitor complex (see right) shows highly slowed dynamics (> 100-fold, [2797], from 20 bridging and 74 interfacial water molecules compared with 459 non-interfacial water molecules, reminiscent of glassy water even at ambient temperatures. Similar effects were noted at protein−DNA interfaces. The strength of these effects is thought due to the presence of a strong electrostatic field, driven by the binding surface’s electrostatic complementarity [2797].

 

Many kinase inhibitors (used as cancer drugs) have recognition mechanisms involving water molecules structured by surrounding protein [2277]. Water's small size, polarity, and conformational flexibility, together with the strength and directionality of the interactions, ensures good fits whilst retaining flexibility and ease of reversibility. The driving force for binding depends not only on the interaction of the biological molecules with each other but the energetic cost for the necessary removal of hydration water and the energetic gain for the subsequent molecular rearrangement of the hydration water molecules [1793, 1805, 2363]. It may well be that the main driving force for the protein-protein interactions is not newly formed electrostatic attractive interactions between the proteins but the entropic gain by the displaced water, as shown for actin-myosin binding [2614]. The use of water may also be useful in broadening the specificity of such links; for example, the peptide-binding protein OppA uses several flexibly-adaptive water molecules to hydrogen bond and shield charges when binding to lysine-X-lysine tripeptides, where X is any one of the twenty common amino acids [1445]. In some cases, structured and structuring water molecules have also been found to be very important for inhibitor recognition [2128]. 

 

[Back to Top to top of page] 

Hydrogen bonding through water enabling molecular recognition Water in protein and enzyme function
  

The energetic optimization of mutual hydrogen-bonded networks between protein, water, and ligand is an intrinsic part of the molecular recognition process in enzymes, binding proteins and biological macromolecules generally [412]. 

Note that water bound and oriented in empty ligand binding sites will reduce the entropy of activation when replaced by the ligand [414]. Similarly, water next to protein surfaces involved in complex formation also shows reduced motion (lower entropy) whereas areas that do not contribute to protein-protein complex formation show increased motion, thus suggesting that evolution of protein surfaces
maximizes the entropy gain arising from the exclusion of hydration water when forming dry protein-protein interfaces [2428]. 

 

Proteins may create aqueous biphasic systems at their interfaces, which can be used to partition materials into, or out of, particular water pools. Many membrane proteins, such as the translocons required for the cotranslational assembly of membrane proteins, contain large water-filled cavities (water pools). For example, the heterotrimeric SecY translocon complex partitions membrane protein molecules by creating a water pool that conducts nascent hydrophobic peptide chains through the translocon during secretion or membrane insertion. These pools contain water that behaves very differently from bulk water having retarded rotational dynamics and aligned dipoles [2446]. 

 

Water molecules are arranged/arrange themselves forming a gradient in water dynamics around 'active' sites such that they facilitate the reactions, via the active site's 'hydration funnel' [2237, 2475]. This arrangement often leads to the localized water molecules being activated for reaction either directly or catalytically. Also, it may lead to small empty spaces (the so-called dehydrons) and basic reactivity of surface water [2164].

A chain of ten water molecules linking the end of one helix to the middle of another in glucoamylase-471 
Figure 1. The water network links secondary structures within the protein and so determines not only the fine detail of the protein's structure but also how particular molecular vibrations may be preferred. The above chain of ten water molecules, linking the end of one α-helix  (helix 9, 211-227) to the middle of another (helix 11, 272-285) is found from the X-ray diffraction data of glucoamylase-471, a natural proteolytic fragment of Aspergillus awamori glucoamylase [155]. 

 

Conserved water binding site in ribonucleases 
Figure 2. The above centrally-placed water molecule makes strong hydrogen bonds to residues in three separated parts of the ribonuclease molecule holding them together. This water molecule and its binding site are conserved across the entire family of microbial ribonucleases [345].
Water molecules have also proved integral to the structure and biological function of a dimeric hemoglobin [377]. The internal water molecules in proteins have been surveyed [725]. They stabilize
proteins by filling internal cavities and by interacting with backbone residues of loops and other polar atoms buried in the protein core and may also act as lubricants to favor loop dynamics [2296]. Many enzyme processes utilize several domains or subunits that fluctuate between several states during the catalysis. These fluctuations involve only low energy structural changes due to the directing and lubricating action of water [2855].

 

The energy barrier in proton-transfer and electron-transfer enzymic reactions may be lowered by creating a large polarizability at the active site due to electrostatic fluctuations of the hydration water [2964]. Both proton-transfer processes [907, 160b] and electron-transfer reactions [908] may be further facilitated by ordered water molecules connecting proton donor to acceptor sites or electron donor to electron acceptor sites respectively. In both cases, the transfer is faster where the linking water molecules possess stronger hydrogen bonding. Further, a network of hydrogen-bonded water molecules plays a catalytic role in water oxidation in spinach photosystem II [1941]. In the cyanobacterial photosystem II, a water channel has been found linking bulk water to the active Mn4 CaO5 cluster and key hydrogen-bond water networks.within the protein's interior; where the supplied water molecules are oxidized [2979].

.

putative water wire
Local water structures and proton-conducting hydrogen-bonded water wires (see left) can form rapidly (~10 ns or so) in response to control by relatively small-scale rearrangements of the protein matrix [2101]. In some cases, the water wire may be affected by the binding of different ions. An example is the Cl− /H+ transporter protein that catalyzes the 2:1 stoichiometrically coupled exchange of Cl− and H+ across biological membranes. The water wire is positioned with the aid of the chloride ion but other ions such as F−, NO3−, and SCN− bind differently and change the stoichiometry or destroy the active wire [2576]. The movements of protons through proton-transport membrane channels are coupled to the diffusion of water through the pores, but these protons cross faster than the diffusion of H3O+. As examples, both the voltage-gated proton channel 1 protein (Hv1) [2849], implicated in diseases including cancer, and the influenza M2 channel [3085], essential for the reproduction of the flu virus, are trans-membrane four-helix bundles that transport protons down water wires. 


Cartoon showing water-assisted asparagine recognition and aspartate discrimination by asparaginyl-tRNA synthetaseFigure 3. Cartoon showing water-assisted asparagine recognition and aspartate discrimination by asparaginyl-tRNA synthetase. (a)  Amongst the bound water molecules in the unliganded enzyme, one (Wat1) is bound between the Leu229 peptide carbonyl and the Glu230 carboxylate and another (Wat4) between the side chains of Tyr333 and Arg364. (b) These water molecules link to the amide group on the asparagine-AMP substrate on binding and two water molecules are released (Wat2 and Wat3). Aspartate-AMP cannot bind at the same site as the Wat1 water molecule is not able to donate three hydrogen bonds. Figure partially redrawn from [2336].

  

The heme–copper ba3-oxidoreductase from Thermus thermophilus uses a highly conserved water molecule (w941), held between two propionate groups as part of a one proton input channel, to deliver protons to the active site for both O2 chemistry and proton pumping (see below, [2322]); w=water, H=histidine, D=aspartate, prop=propionate, Y=tyrosine).

ba3-cytochrome c oxidase , from T. thermophilus, 1EHK
Several water molecules are hydrogen-bonded to the propionate-A–H+ from the heme-a3, and w941 and w946 (as shown above) act as a Zundel cation when protonated. The neighboring histidine (H376, near the 'free' water pool) controls the protonation state of the propionic acid (A) aspartic acid (D372) pair, and hence through another histidine (H283) to the heme-a3 oxidation state [2322].

The catalytic zinc ion of a membrane type I matrix metalloproteinase with a small peptide product (R'), from [3058] 

Water molecules can also be used by enzymes to facilitate the movement of substrates and products towards and away from their active sites. Shown right is the catalytic zinc ion of a membrane type I matrix metalloproteinase with a small peptide product (R') formed immediately after peptide cleavage [3058]. There are two important auxiliary water molecules at this active site. One water molecule (WB, lying in front) acts to stabilize the protonated reaction intermediate by binding to the β-oxygen of the product carboxylate, while the second water molecule (WA, lying behind) facilitates the product release by hydrogen-bonding to between the α-oxygen of the product carboxylate and the glutamate carboxylate. 

[Back to Top to top of page] 


--------------------------------------------------------------------------------

Footnotes
a To avoid such activity loss, proteins generally avoid crystal formation, perhaps by evolutionary design involving surface kosmotropic lysine residues that minimize self-aggregation [771]. [Back]

 

Hydrophobicity: (phenylalanine=isoleucine=leucine)>(tryptophan-cysteine=cystine=valine=methionine)>tyrosine>(alanine=histidine)>(glycine=threonine=proline)>(serine=glutamine=asparagine=arginine)>(aspartic acid=glutamic acid)>lysine 
 

b A historical review of the 'hydrophobic effect' in proteins is available [1393].

 

An approximate comparison of the hydrophobicity of the amino acids is given in the table opposite. Further discussion of relative hydrophobicities is given elsewhere as is a classification of hydrophobicity scales [633]. It should be noted that such hydrophobic interactions are particularly important in stabilizing interdomain and quaternary interactions. [Back]

c The effect of strongly-bound surface water molecules on preventing protein-protein interactions is described in [881]. [Back]

 

d Alternatively to this molecular group approach, some of this increased density may be attributed to electrostriction pressure (that is, local pressure increase due to the localized electric field) [951]. [Back]

 

e Other techniques, such as NMR, are incapable of showing this. [Back] 

 

f Other research shows somewhat less water affected out to only 0.85 nm; 3-4 layers [2848]. [Back]

